packages/devtools_app/test/memory/framework/offline_data_test.dart

```diff
@@ -15,47 +17,61 @@ import 'package:devtools_app/src/shared/primitives/simple_items.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
-  test(
-    '$OfflineMemoryData serializes and deserializes correctly',
-    () {
-      final item = OfflineMemoryData(
-        DiffPaneController(loader: null, rootPackage: 'root'),
-        ProfilePaneController(
-          mode: ControllerCreationMode.connected,
-          rootPackage: 'root',
-        ),
-        ChartData(
-          mode: ControllerCreationMode.offlineData,
-          isDeviceAndroid: true,
-          timeline: MemoryTimeline(),
-          interval: ChartInterval.theDefault,
-          isLegendVisible: true,
-        ),
-        ClassFilter.empty(),
-        selectedTab: 0,
-      );
+  for (final encode in [true, false]) {
+    test(
+      '$OfflineMemoryData serializes and deserializes correctly',
```

Comment:
add the value of `encode` to this string so that can differentiate the test cases